src/org/parosproxy/paros/extension/option/OptionsCertificatePanel.java

```diff
@@ -59,7 +59,6 @@
 import org.parosproxy.paros.model.Model;
 import org.parosproxy.paros.model.OptionsParam;
 import org.parosproxy.paros.view.AbstractParamPanel;
-import org.parosproxy.paros.view.View;

```

Comment:
It seems the branch needs to be rebased, this was already done in a previous commit.